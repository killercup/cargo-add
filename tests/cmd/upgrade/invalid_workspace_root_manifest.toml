bin.name = "cargo-upgrade"
args = ["upgrade", "--workspace"]
status.code = 1
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
