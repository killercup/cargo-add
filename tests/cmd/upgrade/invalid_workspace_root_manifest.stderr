Command failed due to unhandled error: Failed to get workspace metadata

Caused by: `cargo metadata` exited with an error: error: failed to parse manifest at `[CWD]/Cargo.toml`

Caused by:
  could not parse input as TOML

Caused by:
  expected an equals, found an identifier at line 1 column 6
